.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

```suggestion
node_modules
.env
```
Don't forget your env file!